### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/how-to/general.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/how-to/general.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/how-to/general.md)

Discussion: #9193

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit d7103709b3a47843d95b6a459f57bee65a291a6e at: https://d99d2760.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit ca6dd224daa11a4d4b7cdeb79c61bae5983795dd at: https://6af2593e.fastapitiangolo.pages.dev

File: docs/tr/docs/how-to/general.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Bu sayfada genel veya sıkça sorulan sorular için dokümantasyonun diğer yerlerine yönlendirmeler bulunmaktadır.
+
+## Veriyi Filtreleme - Güvenlik
+
+Döndürmeniz gereken veriden fazlasını döndürmemenizi sağlamak için, [Tutorial - Response Model - Return Type](../tutorial/response-model.md){.internal-link target=_blank} sayfasını okuyun.
+
+## Dokümantasyon Etiketleri - OpenAPI
+
+*Yol işlemleri*ni dokümantasyon arayüzünde gruplamak ve etiketlemek için, [Tutorial - Path Operation Configurations - Tags](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} sayfasını okuyun.
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:11]:
```suggestion
*Yol operasyonlarını* dokümantasyon arayüzünde gruplamak ve etiketlemek için, [Tutorial - Path Operation Configurations - Tags](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} sayfasını okuyun.
```

File: docs/tr/docs/how-to/general.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Dokümantasyon Özeti ve Açıklaması - OpenAPI
+
+*Yol işlemleri*nize özet ve açıklama eklemek ve bunları dokümantasyon arayüzünde göstermek için, [Tutorial - Path Operation Configurations - Summary and Description](../tutorial/path-operation-configuration.md#summary-and-description){.internal-link target=_blank} sayfasını okuyun.
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:15]:
```suggestion
*Yol operasyonlarınıza* özet ve açıklama eklemek ve bunları dokümantasyon arayüzünde göstermek için, [Tutorial - Path Operation Configurations - Summary and Description](../tutorial/path-operation-configuration.md#summary-and-description){.internal-link target=_blank} sayfasını okuyun.
```

File: docs/tr/docs/how-to/general.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Dokümantasyon Yanıt Açıklaması - OpenAPI
+
+Dokümantasyon arayüzünde gösterilen yanıt açıklamasını tanımlamak için, [Tutorial - Path Operation Configurations - Response description](../tutorial/path-operation-configuration.md#response-description){.internal-link target=_blank} sayfasını okuyun.
+
+## *Yol İşlemi*ni Kullanımdan Kaldırma - OpenAPI
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:21]:
```suggestion
## *Yol Operasyonunu* Kullanımdan Kaldırma - OpenAPI
```

**Review by hasansezertasan — COMMENTED** [review]
Follow the convention

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9de87ac638da3d805d5f28f442d7764d307787eb at: https://f09cd66b.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit eaef54ad7e32c3a943132087777ff99a7214a6c9 at: https://ac9678ba.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

File: docs/tr/docs/how-to/general.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Genel - Nasıl Yapılır - Tarifler
+
+Bu sayfada genel veya sıkça sorulan sorular için dokümantasyonun diğer yerlerine yönlendirmeler bulunmaktadır.
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:3]:
```suggestion
Bu sayfada genel ve sıkça sorulan sorular için dokümantasyonun diğer sayfalarına yönlendirmeler bulunmaktadır.
```

File: docs/tr/docs/how-to/general.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Genel - Nasıl Yapılır - Tarifler
+
+Bu sayfada genel ve sıkça sorulan sorular için dokümantasyonun diğer sayfalarına yönlendirmeler bulunmaktadır.
+
+## Veriyi Filtreleme - Güvenlik
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:5]:
```suggestion
## Veri Filtreleme - Güvenlik
```

File: docs/tr/docs/how-to/general.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# Genel - Nasıl Yapılır - Tarifler
+
+Bu sayfada genel ve sıkça sorulan sorular için dokümantasyonun diğer sayfalarına yönlendirmeler bulunmaktadır.
+
+## Veri Filtreleme - Güvenlik
+
+Döndürmeniz gereken veriden fazlasını döndürmemenizi sağlamak için, [Tutorial - Response Model - Return Type](../tutorial/response-model.md){.internal-link target=_blank} sayfasını okuyun.
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:7]:
```suggestion
Döndürmeniz gereken veriden fazlasını döndürmediğinizden emin olmak için, [Tutorial - Response Model - Return Type](../tutorial/response-model.md){.internal-link target=_blank} sayfasını okuyun.
```

File: docs/tr/docs/how-to/general.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+Bu sayfada genel ve sıkça sorulan sorular için dokümantasyonun diğer sayfalarına yönlendirmeler bulunmaktadır.
+
+## Veri Filtreleme - Güvenlik
+
+Döndürmeniz gereken veriden fazlasını döndürmediğinizden emin olmak için, [Tutorial - Response Model - Return Type](../tutorial/response-model.md){.internal-link target=_blank} sayfasını okuyun.
+
+## Dokümantasyon Etiketleri - OpenAPI
+
+*Yol operasyonlarını* dokümantasyon arayüzünde gruplamak ve etiketlemek için, [Tutorial - Path Operation Configurations - Tags](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} sayfasını okuyun.
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:11]:
```suggestion
*Yol operasyonlarınıza* etiketler ekleyerek dokümantasyon arayüzünde gruplar halinde görünmesini sağlamak için, [Tutorial - Path Operation Configurations - Tags](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} sayfasını okuyun.
```

File: docs/tr/docs/how-to/general.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+Döndürmeniz gereken veriden fazlasını döndürmediğinizden emin olmak için, [Tutorial - Response Model - Return Type](../tutorial/response-model.md){.internal-link target=_blank} sayfasını okuyun.
+
+## Dokümantasyon Etiketleri - OpenAPI
+
+*Yol operasyonlarınıza* etiketler ekleyerek dokümantasyon arayüzünde gruplar halinde görünmesini sağlamak için, [Tutorial - Path Operation Configurations - Tags](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} sayfasını okuyun.
+
+## Dokümantasyon Özeti ve Açıklaması - OpenAPI
+
+*Yol operasyonlarınıza* özet ve açıklama eklemek ve bunları dokümantasyon arayüzünde göstermek için, [Tutorial - Path Operation Configurations - Summary and Description](../tutorial/path-operation-configuration.md#summary-and-description){.internal-link target=_blank} sayfasını okuyun.
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:15]:
```suggestion
*Yol operasyonlarınıza* özet ve açıklama ekleyip dokümantasyon arayüzünde görünmesini sağlamak için, [Tutorial - Path Operation Configurations - Summary and Description](../tutorial/path-operation-configuration.md#summary-and-description){.internal-link target=_blank} sayfasını okuyun.
```

File: docs/tr/docs/how-to/general.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+## Dokümantasyon Etiketleri - OpenAPI
+
+*Yol operasyonlarınıza* etiketler ekleyerek dokümantasyon arayüzünde gruplar halinde görünmesini sağlamak için, [Tutorial - Path Operation Configurations - Tags](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} sayfasını okuyun.
+
+## Dokümantasyon Özeti ve Açıklaması - OpenAPI
+
+*Yol operasyonlarınıza* özet ve açıklama ekleyip dokümantasyon arayüzünde görünmesini sağlamak için, [Tutorial - Path Operation Configurations - Summary and Description](../tutorial/path-operation-configuration.md#summary-and-description){.internal-link target=_blank} sayfasını okuyun.
+
+## Dokümantasyon Yanıt Açıklaması - OpenAPI
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:17]:
```suggestion
## Yanıt Açıklaması Dokümantasyonu - OpenAPI
```

File: docs/tr/docs/how-to/general.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+*Yol operasyonlarınıza* etiketler ekleyerek dokümantasyon arayüzünde gruplar halinde görünmesini sağlamak için, [Tutorial - Path Operation Configurations - Tags](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} sayfasını okuyun.
+
+## Dokümantasyon Özeti ve Açıklaması - OpenAPI
+
+*Yol operasyonlarınıza* özet ve açıklama ekleyip dokümantasyon arayüzünde görünmesini sağlamak için, [Tutorial - Path Operation Configurations - Summary and Description](../tutorial/path-operation-configuration.md#summary-and-description){.internal-link target=_blank} sayfasını okuyun.
+
+## Yanıt Açıklaması Dokümantasyonu - OpenAPI
+
+Dokümantasyon arayüzünde gösterilen yanıt açıklamasını tanımlamak için, [Tutorial - Path Operation Configurations - Response description](../tutorial/path-operation-configuration.md#response-description){.internal-link target=_blank} sayfasını okuyun.
```
Comment (hasansezertasan) [inline docs/tr/docs/how-to/general.md:19]:
```suggestion
Dokümantasyon arayüzünde yer alan yanıt açıklamasını tanımlamak için, [Tutorial - Path Operation Configurations - Response description](../tutorial/path-operation-configuration.md#response-description){.internal-link target=_blank} sayfasını okuyun.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8f6d3061569c9cbba0b64399330d8fa8beb2a489 at: https://20f7e42f.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

**Comment by alejsdev** [comment]
Great. Thanks @hasansezertasan :nerd_face: 

And thanks for the help @OzgunCaglarArslan :100:

**Pull request merged**